### ISSUE TITLE
dependency failed to start: container esk-es02-1 exited (1)

### ISSUE THREAD

**Issue by MikzTwix** [PacktPublishing/Practical-Detection-Engineering#3]
Someone please help. I received the error "dependency failed to start: container esk-es02-1 exited (1)." I made no changes to the docker-compose file except for the volume and cert path, but the error persists.

**Comment by deepaliprasad** [comment]
I am facing a similar error today with `dependency failed to start: container esk-es01-1 exited (1)`
Any suggestions on what could be going wrong?

**Comment by cogphn** [comment]
can you save the logs for that container and send it here? You can get the logs by using the command `docker compose logs es02`.

**Comment by mvani89** [comment]
Was this ever addressed? I am working through this right now and getting the same error.

**Comment by MikzTwix** [comment]
Nope, I just deploy GOAD with ELK and that is the lab that I have been using right now. Cause that is more straightforward. Rather than this lab from the book.

**Comment by mvani89** [comment]
Thanks for the quick update. I'm going to spend a couple more hours trying to work through this. I have another actual ELK deployment about 3/4 setup, maybe Ill run with that. Ill post back if I can figure it out.

**Comment by madhavanrx18** [comment]
I figured out the issue, it was in the .env file , change the `STACK_VERSION` to 8.14.3.
